### PR TITLE
[11.0] mig partner contact gender

### PR DESCRIPTION
**Pull request by NL66278**
Instead of (or on top on) https://github.com/OCA/partner-contact/pull/510 which seems to be abandoned by the author.

### CODE REVIEW

File: partner_contact_gender/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Copyright 2014-2015 Grupo ESOC <www.grupoesoc.es>
+# Copyright 2017-Apertoso N.V. (<http://www.apertoso.be>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Contact gender",
+    "summary": "Add gender field to contacts",
+    'version': '11.0.1.0.0',
+    "category": "Customer Relationship Management",
+    "website": "https://www.tecnativa.com/",
```
Comment (nikul-serpentcs) [inline partner_contact_gender/__manifest__.py]: @NL66278 Replace URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)

File: partner_contact_gender/views/res_partner.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_personal_information_gender" model="ir.ui.view">
+        <field name="name">Partner gender: personal info</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="partner_contact_personal_information_page.personal_information"/>
+        <field name="arch" type="xml">
+            <data>
```
Comment (nikul-serpentcs) [inline partner_contact_gender/views/res_partner.xml]: `<data>` not need

**Review by nikul-serpentcs — COMMENTED** [review]
Minor Change

**Comment by NL66278** [comment]
@nikul-serpentcs I applied the changes you requested.

**Review by nikul-serpentcs — APPROVED** [review]
If this https://github.com/OCA/partner-contact/pull/662#issuecomment-441590177 Fine then Code review LGTM

File: partner_contact_gender/hooks.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2016-2018 Therp BV <https://therp.nl>.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo import api, SUPERUSER_ID
+
+
+def post_init_hook(cr, pool):
```
Comment (gfcapalbo) [inline partner_contact_gender/hooks.py]: replace "pool" with "registry" in V11 style , also consider moving post_init_hook in module  __init__  for a more compact layout (as done in most v11 core modules)

**Review by gfcapalbo — CHANGES_REQUESTED** [review]

File: partner_contact_gender/hooks.py (hunk 3 above)
Comment (NL66278) [reply partner_contact_gender/hooks.py]: @gfcapalbo I will replace the argument name pool with registry. However including the hook code into __init__.py itself is not what the OCA conventions say:
https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#installation-hooks

**Review by gfcapalbo — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Can you please make some commit squashing before merging?

At least adjacent "OCA Transbot" commits, but also useless adjacent commits coming from same contributor that are in this PR.

**Comment by NL66278** [comment]
@pedrobaeza I squashed what I could ;-)

**Comment by NL66278** [comment]
Is there any reason why this PR is not merged? As it has been approved and even carries the 'Ready to merge' label....

**Comment by pedrobaeza** [comment]
Reason is time for doing it :wink: , as I still want to remove useless commits like "[MIG] Make modules uninstallable", and I don't want to charge you with that. I'll do it today.

**Comment by NL66278** [comment]
@pedrobaeza Thank you very much!

**Pull request merged**